src/rest_api/classes/phenotype/core.clj

```diff
@@ -3,46 +3,266 @@
    [clojure.string :as str]
    [datomic.api :as d]
    [pseudoace.utils :as pace-utils]
+   [rest-api.classes.paper.core :as paper-core]
+   [rest-api.formatters.object :as obj]
+   [rest-api.formatters.object :as obj :refer  [pack-obj]]))
 
+(defn- create-pato-term [id label entity-term entity-type pato-term]
+  (let [pato-id (str/join "_" [id label pato-term])]
+    {pato-id
+     {:pato_evidence
+      {:entity_term entity-term
+       :entity_type label
+       :pato_term pato-term}
+      :key pato-id}}))
 
-(defn- phenotype-table [db gene not-observed?]
-  (let [var-phenos (into {} (d/q (if not-observed?
-                                   q-gene-var-not-pheno
-                                   q-gene-var-pheno)
-                                 db gene))
-        rnai-phenos (into {} (d/q (if not-observed?
-                                    q-gene-rnai-not-pheno
-                                    q-gene-rnai-pheno)
-                                  db gene))
-        phenos (set (concat (keys var-phenos)
-                            (keys rnai-phenos)))]
-    (->>
+(defn get-pato-from-holder [holder]
+  (let [sot (for [eq-annotations {"anatomy-term" "anatomy-term"
+                                  "life-stage" "life-stage"
+                                  "go-term" "go-term"
+                                  "molecule-affected" "molecule"}
+                  :let [[eq-key label] eq-annotations]]
+              (for [eq-term ((keyword "phenotype-info" eq-key) holder)]
+                (let [make-pi-kw (partial keyword
+                                       (str "phenotype-info." eq-key))
+                      pato-term-kw (make-pi-kw "pato-term")
```

Comment:
Formatting in this let expression seems a bit off when call to function spans > 1 line